src/app/(app)/(home)/navbar.tsx

```
@@ -0,0 +1,64 @@
+    <nav className="h-20 flex border border-b justify-between font-medium bg-white pl-5">
+      <Link href={"/"} className="pl-6 flex items-center">
+        <span className={`${poppins.className} text-5xl font-semibold`}>
+          bunroad
+        </span>
+      </Link>
+
+      <div className="items-center justify-center gap-2 hidden lg:flex">
+        {navbarItems.map((item) => (
+          <NavbarItem
+            href={item.href}
+            children={item.children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)